src/libecalc/presentation/exporter/queries.py

```diff
@@ -202,8 +203,103 @@ def query(
                         emissions = installation_graph.get_emissions(venting_emitter.id)
 
                         for emission_name, emission in emissions.items():
+                            rate = emission.rate
+
+                            emission_volumes = TimeSeriesRate.from_timeseries_stream_day_rate(
+                                rate, regularity=regularity
+                            ).to_volumes()
+                            unit_in = emission_volumes.unit
+                            for timestep, emission_volume in emission_volumes.datapoints():
+                                if self.emission_type is None or emission_name == self.emission_type:
+                                    aggregated_result[timestep] += emission_volume
+
+            if aggregated_result:
+                sorted_result = dict(dict(sorted(zip(aggregated_result.keys(), aggregated_result.values()))).items())
+                sorted_result = {**dict.fromkeys(installation_time_steps, 0.0), **sorted_result}
+                date_keys = list(sorted_result.keys())
```

Comment:
This is done for quite some time ago:

- Sort dictionary on dates
- Adding final date to dictionary. E.g. if it is only one year: 2027 - 2028. The aggregated result will only contain value for 2027. 2028 is added with value 0, for TimeSeriesVolume to handle it correctly. 

This can probably be done easier: just sort dictionary, then feed sorted_result into TimeSeriesVolume, together with installation_time_steps (instead of date_keys). However, maybe take this in another PR?